src/components/Input/index.js

```diff
@@ -32,11 +33,11 @@ function Input({
         className
       })}
       type={search ? "search" : type}
+      name={name}
       onChange={(e) => {
         e.preventDefault()
         onChange(e.target.value)
       }}
-      value={value}
```

Comment:
value should not be removed in case we want to pass an initial value to the input